integration-cli/daemon/daemon.go

```diff
@@ -718,7 +718,7 @@ func (d *Daemon) ReloadConfig() error {
 	errCh := make(chan error)
 	started := make(chan struct{})
 	go func() {
-		_, body, err := request.Get(d.Sock(), "/events")
+		_, body, err := request.DoOnHost(d.Sock(), "/events", request.Method(http.MethodGet))
```

Comment:
@thaJeztah probably yes, but `request.SockRequest` is deprecated and I want to get rid of it 👼